postfix/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,74 @@
+name: Postfix
+files:
+- name: postfix.yaml
+  options:
+  - template: init_config
+    options:
+    - name: postfix_user
+      required: true
+      enabled: true
+      description: |
+        The user running dd-agent must have passwordless sudo access for the find
+        command to run the postfix check.  Here's an example:
+        example /etc/sudoers entry:
+          dd-agent ALL=(postfix) NOPASSWD:/usr/bin/find /var/spool/postfix/incoming -type f
+
+        Redhat/CentOS/Amazon Linux flavours need to add:
+          Defaults:dd-agent !requiretty
+      value:
+        type: string
+        example: postfix
+    - name: postqueue
+      description: |
+        Set `postqueue: true` to gather mail queue counts using `postqueue -p` without the use of sudo.
+        Postqueue binary is ran with set-group ID privileges, so that it can connect to Postfix daemon processes.
+        Only `tags` keys are used from `instances` definition.
+        Postfix has internal access controls that limit activities on the mail queue. By default,
+        Postfix allows `anyone` to view the queue. On production systems where the Postfix installation may be
+        configured with stricter access controls, you may need to grant the dd-agent user access to view the mail queue.
+
+        postconf -e "authorized_mailq_users = dd-agent"
+
+        http://www.postfix.org/postqueue.1.html
+
+        authorized_mailq_users (static:anyone)
+        List of users who are authorized to view the queue.
```

Comment:
Unclear to me what these lines are saying. Do they run the authorized_mailq_users command to get a list of users?

Reply:
I think it means that authorized_mailq_users is the list of users who are authorized to view the queue. I'll reformat that sentence